configure.ac

```diff
@@ -2243,8 +2243,11 @@ dnl struct {char *name; int *addr; char *doc;} desc = spam(foo, "something");
 dnl ]], [[;]])],[cpp_type=ansi],[AC_DEFINE(HAVE_OLD_CPP) cpp_type=traditional])
 dnl AC_MSG_RESULT($cpp_type)
 
+dnl backwards compatibility: autoconf 2.71 deprecates STDC_HEADERS
+dnl assume C99 compilers provide ANSI C headers
+AC_DEFINE(STDC_HEADERS, 1, [Define to 1 if you have the ANSI C header files.])
```

Comment:
~Why do we need to keep this? It is no longer used in the code base.~
Aaah, it is used in `libffi_osx` in `_ctypes`.